README.md

```diff
@@ -102,10 +114,21 @@ This extension provides 4 actions:
     ```php
     <?php
 
+    namespace app\controllers;
+
+    use Yii;
+    use yii\web\Controller;
+    use yii\helpers\Url;
+
+
     class SamlController extends Controller {
 
+        // Remove CSRF protection
+        public $enableCsrfValidation = false;
```

Comment:
it should work, but this will be more complex. Since the SamlController is something that the dev need to implement, we should keep it as simple as possible.